test/ctia/entity/actor_test.clj

```diff
@@ -51,7 +51,7 @@
                 app
                 (assoc new-actor-maximal :title "foo")
                 :actors
-                345
```

Comment:
345 -> 10 is quite a big difference... worst-case scenario, this could mean that some implicit coverage is being lost

Did you assess the test contents themselves?

(I realise that may be hard)

Reply:
AFAICT [ctia.test-helpers.pagination/total->limit](https://github.com/threatgrid/ctia/blob/75fa716fb30579a382b9da71b37b76e120b1c40a/test/ctia/test_helpers/pagination.clj#L8) compensates for total pagination size. I think we only ever use an offset of `1`, which I think means a pagination of 10 would probably work.

Reply:
There are other places where I changed the sample size from 5 to 3. That's probably pushing it, I'll revert that.

Reply:
I added a couple of assertions in `ctia.test-helpers.pagination` to make sure the pagination limit is always positive.

Reply:
>  I think we only ever use an offset of 1

In case it helps, you might want to try a one-off assertion verifying that